src/internal/observable/ConnectableObservable.ts

```diff
@@ -12,8 +12,8 @@ import { refCount as higherOrderRefCount } from '../operators/refCount';
 export class ConnectableObservable<T> extends Observable<T> {
 
   protected _subject: Subject<T>;
-  protected _refCount: number = 0;
-  protected _connection: Subscription;
+  private _refCount: number = 0;
+  private _connection: Subscription;
```

Comment:
This is technically a breaking change for TypeScript users.

Reply:
Yeah, as this would be a breaking change, I don't see any advantage to changing these to `private`. Especially as this is a 'perf' PR. This is unrelated to perf.

Reply:
reverted to protected